### PR TITLE
Admin home page

### PR DESCRIPTION
**Pull request by Saya221**
image

### CODE REVIEW

**Comment by Saya221** [comment]
Ready

**Comment by Saya221** [comment]
Ready

File: app/views/layouts/_sidebar_admin.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <ul class="dropdown-menu" role="menu">
+          <li class="dropdown-header"></li>
+          <li><%= link_to "1", "#"%></li>
+          <li><%= link_to "2", "#"%></li>
+          <li><%= link_to "3", "#"%></li>
+          <li><%= link_to "4", "#"%></li>
+          <li><%= link_to "5", "#"%></li>
+        </ul>
+      </li>
+        <li><%= link_to t(".help"), "#"%></li>
```
Comment (phongnvk-0916) [inline app/views/layouts/_sidebar_admin.html.erb]: indent

**Comment by Saya221** [comment]
Ready

**Pull request merged**